CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.21.3...master
 
+### Fixed
+- Upgrade to Ubuntu 20.04 Focal in `StackVirtualUsersFormula`.
```

Comment:
Why not mention it fixes a deadlock? This changelog says nothing about what has been fixed.

Reply:
I will update the changelog in a different PR